src/sentry/static/sentry/app/views/organizationDiscover/result/vizualizationsDropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import DropdownLink from 'app/components/dropdownLink';
+import MenuItem from 'app/components/menuItem';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import theme from 'app/utils/theme';
+
+class VisualizationsDropdown extends React.PureComponent {
```

Comment:
We don't tend to use PureComponent vs Component in cases like this... is there a specific reason we'll want to use it here?

Reply:
I noticed PureComponent was used in other uses of dropdownLink, like in the `sortOptions.jsx` 
It seemed useful because it does a shallow comparison of props when they change and we rely on `props.visualization`, but there doesn't seem to be any difference when I replaced it with Component.